pkg/controller.v1/tensorflow/tfjob_controller.go

```diff
@@ -452,32 +452,20 @@ func (r *TFJobReconciler) UpdateJobStatus(job interface{}, replicas map[kubeflow
 		if ContainsChiefOrMasterSpec(tfJob.Spec.TFReplicaSpecs) {
 			if kubeflowv1.IsChieforMaster(rtype) {
 				if running > 0 {
-					msg := fmt.Sprintf("TFJob %s/%s is running.",
-						tfJob.Namespace, tfJob.Name)
-					err := commonutil.UpdateJobConditions(jobStatus, kubeflowv1.JobRunning, corev1.ConditionTrue,
-						commonutil.NewReason(kubeflowv1.TFJobKind, commonutil.JobRunningReason), msg)
-					if err != nil {
-						commonutil.LoggerForJob(tfJob).Infof(
-							"Append tfjob condition error: %v", err)
-						return err
-					}
+					msg := fmt.Sprintf("TFJob %s/%s is running.", tfJob.Namespace, tfJob.Name)
+					commonutil.UpdateJobConditions(jobStatus, kubeflowv1.JobRunning, corev1.ConditionTrue, commonutil.NewReason(kubeflowv1.TFJobKind, commonutil.JobRunningReason), msg)
 				}
-				if expected == 0 {
-					msg := fmt.Sprintf("TFJob %s/%s successfully completed.",
-						tfJob.Namespace, tfJob.Name)
-					r.recorder.Event(tfJob, corev1.EventTypeNormal, commonutil.NewReason(kubeflowv1.TFJobKind, commonutil.JobSucceededReason), msg)
-					if jobStatus.CompletionTime == nil {
-						now := metav1.Now()
-						jobStatus.CompletionTime = &now
-					}
-					err := commonutil.UpdateJobConditions(jobStatus,
-						kubeflowv1.JobSucceeded, corev1.ConditionTrue, commonutil.NewReason(kubeflowv1.TFJobKind, commonutil.JobSucceededReason), msg)
-					if err != nil {
-						commonutil.LoggerForJob(tfJob).Infof("Append tfjob condition error: %v", err)
-						return err
-					}
-					trainingoperatorcommon.SuccessfulJobsCounterInc(tfJob.Namespace, r.GetFrameworkName())
+			}
+			if expected == 0 {
```

Comment:
It seems the nesting of `if exepected == 0 {` is changed. Any reason to move this?

Reply:
Oops. This is my mistake. It's a good catch! Thanks!

Reply:
Done.